json/src/spec/spec.rs

```diff
@@ -31,6 +31,7 @@ pub enum ForkSpec {
 	Byzantium,
 	Constantinople,
 	ConstantinopleFix,
+	Istanbul,
```

Comment:
@ordian This enum throws warnings for each variant. :/ Either we add dummy docs or annotate it `#[allow(missing_docs)]`.

Reply:
it is probably my fault in #11027